dlt/extract/schema.py

```diff
@@ -67,7 +67,12 @@ def table_schema(self, item: TDataItem =  None) -> TPartialTableSchema:
         resolved_template.pop("incremental", None)
         table_schema = self._merge_keys(resolved_template)
         table_schema["resource"] = self._name
-        validate_dict(TPartialTableSchema, table_schema, f"new_table/{self._name}")
+        validate_dict(
+            spec=TPartialTableSchema,
+            doc=table_schema,
+            path=f"new_table/{self._name}",
+            filter_f=lambda k: not k.startswith("x-")
```

Comment:
would be good to define this prefix ie. in schema/typings

Reply:
Could you please elaborate on this?

Reply:
@rudolfix how about

```
validate_dict_ignoring_xkeys = functools.partial(
    validate_dict,
    filter_f=lambda k: not k.startswith("x-")
)
```

Reply:
OK I know what you mean. since we use "x-" prefix in several places I though that we can define it as constant in schema/typing.py
or better: the whole lambda could be reused